templates/eks/components/eks-database-deploy-04.yml

```diff
@@ -38,7 +38,7 @@ Resources:
       DBSubnetGroupName:
         Ref: DBSubnetGroup
       Engine: postgres
-      EngineVersion: '11.2'
+      EngineVersion: '11.8'

```

Comment:
Is this change just a feature or there were some errors with EngineVersion 11.2?